bitnami/phpmyadmin/values.yaml

```diff
@@ -228,3 +228,5 @@ metrics:
   ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
   ##
   resources: {}
+
+mariadb: {}
```

Comment:
`phpmyadmin` will use default values for mariadb subchart that means it doesn't have `mariadb: {}`, In order to use mariadb helper for passwords I need to add it because it is expected to be there.